src/pages/user/growth.jsx

```
@@ -89,241 +101,71 @@ const growth = () => {
+                  src={activity.image}
+                  alt={activity.title}
+                  className="rounded-md hidden md:block m-2 object-fit px-4"
+                />
+                <div className="flex flex-col">
+                  <CardContent className="text-base pb-3 text-justify">
+                    <p>{activity.content}</p>
+                  </CardContent>
+                </div>
+              </Card>
+            </div>
+          ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)